solutions/src/main/java/ru/sberbank/school/task08/KryoSerializer.java

```diff
@@ -0,0 +1,43 @@
+package ru.sberbank.school.task08;
+
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.Serializer;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KryoSerializer extends Serializer<MapState<GameObject>> {
+
+    @Override
+    public void write(Kryo kryo, Output output, MapState<GameObject> object) {
+        output.write(object.getGameObjects().size());
+        output.writeString(object.getName());
+
+        for (GameObject go : object.getGameObjects()) {
+            output.writeString(go.getType().name());
+            output.writeString(go.getStatus().name());
+            output.writeLong(go.getHitPoints());
+        }
+
+    }
+
+    @Override
+    public MapState<GameObject> read(Kryo kryo, Input input, Class<? extends MapState<GameObject>> type) {
+        int size = input.readInt();
+        String name = input.readString();
+        List<GameObject> objectsList = new ArrayList<>();
+        for (int i = 0; i<size; i++) {
```

Comment:
WhitespaceAround: &#39;&lt;&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-768900445>*